<script>
 debugger
 Promise.resolve("Hola").
  then(s => document.write(
   `Éxito 1: ${s}<br>`)).
  catch(e => document.write(
   `Fallo 1: ${e.message}<br>`))
 Promise.reject(new Error("ouch")).
  then(s => document.write(
   `Éxito 2: ${s}<br>`)).
  catch(e => document.write(
   `Fallo 2: ${e.message}<br>`))
</script>